sbin/build.sh

```diff
@@ -565,6 +565,19 @@ removingUnnecessaryFiles() {
     rm -rf "${dirToRemove}"/demo || true
   fi
 
+  # Generate java.se adopt bundle
+  moduleList="java.se"
+  case "${BUILD_CONFIG[OS_KERNEL_NAME]}" in
+    "darwin") jlink="${jdkTargetPath}/Contents/Home/bin/jlink" ;;
+    *) jlink="${jdkTargetPath}/bin/jlink" ;;
```

Comment:
Windows?  Or does cygwin paths cover this?

Reply:
cygwin paths handles this